webpack/assets/javascripts/react_app/redux/actions/common/index.js

```diff
@@ -11,5 +11,5 @@ export const ajaxRequestAction = ({
   dispatch({ type: requestAction, payload: item });
   return API.get(url)
     .then(({ data }) => dispatch({ type: successAction, payload: { ...item, ...data } }))
-    .catch(error => dispatch({ type: failedAction, payload: { error, item } }));
+    .catch(error => dispatch({ type: failedAction, payload: { item, error: error.message } }));
```

Comment:
does it work with the all components that use `ajaxRequestAction`?

Reply:
Yes, it does. I did do manual testing if that helps.
Basically, there are 4 components that are doing API calls:
BreadcrumbBar, Bookmarks, Host's PowerStatus and Statistics.
3 out of them are using `ajaxRequestAction` - [BreadcrumbBar](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarActions.js) doesn't use `ajaxRequestAction` probably because of `payload` structure nuance.

Bookmarks doesn't use any element set in `item` but does expect `error` to be the error message - which is fixed in this patch.
Host's PowerStatus and Statistics however are expecting the payload to have `id` given in `item` (I guess `matan` missed that change when he pushed [this](https://github.com/theforeman/foreman/commit/ee7002f8f2bffb3a589c28ea1b35460b6bdbf3c0#diff-87d65aaa5cc9106d1b99118725b9b87cL14)) so I decided to keep `item` there and use it in reducers.

Reply:
I guess it would be nicer if I keep the order, though (for a readable diff):
```
payload: { error: error.message, item }
```